services/dashboard/views/apps/streamingapp/streamingapp.js

```diff
@@ -101,9 +99,9 @@ angular.module('dashboard')
       $scope.sendThroughputMetricsDescription = 'Messages are sent from external data source';
       $scope.receiveThroughputMetricsCaption = 'Sink Processors Receive Throughput';
       $scope.receiveThroughputMetricsDescription = 'Messages are received by external data sink';
+      $scope.messageLatencyMetricsCaption = 'Critical Path Latency';
+      $scope.messageLatencyMetricsDescription = 'The total latency on the critical path';
```

Comment:
Suggest to change "Critical Path Latency" to more user friendly name "end to end latency", and in the notes document how end to end latency is caluculated (by critical path)